scripts/install

```diff
@@ -119,15 +119,25 @@ ynh_exec_warn_less ynh_install_composer --phpversion="$phpversion" --workdir="$f
 ynh_script_progression --message="Modifying a config file..."
 
 config="$final_path/.env"
-cp ../conf/.env "$config"
-
-ynh_replace_string --match_string="random_key" --replace_string="$random_key" --target_file="$config"
-ynh_replace_string --match_string="yunouser" --replace_string="$db_name" --target_file="$config"
-ynh_replace_string --match_string="yunopass" --replace_string="$db_pwd" --target_file="$config"
-ynh_replace_string --match_string="yunobase" --replace_string="$db_name" --target_file="$config"
-ynh_replace_string --match_string="yunomail" --replace_string="$email" --target_file="$config"
-ynh_replace_string --match_string="yunodomain" --replace_string="$domain" --target_file="$config"
-ynh_replace_string --match_string="yunoappurl" --replace_string="$domain$path_url" --target_file="$config"
+cp -f "$final_path/.env.example" "$config"
```

Comment:
Why no more using `../conf/.env` ?

Reply:
Taking .env from the sources itself, rather then from ../conf/.env. Is it not better to get it from sources directly?

Reply:
using ../conf/.env allow you to have a pre-filled .env so less `ynh_string_replace`

Reply:
Is it ok now?